08-projects.Rmd

```diff
@@ -6497,7 +6487,7 @@ https://www.uvm.edu/femc/data/archive/project/federal-forest-inventory-analysis-
 - The output of your code piped to `head`.
 ```
 
-##### 3. `fread` is a "Fast and Friendly File Finagler". It is part of the very popular `data.table` package in R. We will learn more about this package next semester. For now, read the documentation [here](https://www.rdocumentation.org/packages/data.table/versions/1.12.8/topics/fread) and use the `cmd` argument in conjunction with your bash code from (2) to preprocess data prior to reading it into a `data.table` in your R environment. 
+##### 3. `fread` is a "Fast and Friendly File Finagler". It is part of the very popular `data.table` package in R. We will learn more about this package next semester. For now, read the documentation [here](https://www.rdocumentation.org/packages/data.table/versions/1.12.8/topics/fread) and use the `cmd` argument in conjunction with your bash code from (2) to read the data of `STATECD`, `LAT`, and `LON` into a `data.table` in your R environment. 
```

Comment:
I find "preprocess" too general a term when first attempting this question. I was wondering what pre-process steps were required. After comparing the solution, I modified the sentence as above.

Reply:
I suppose I considered it preprocessing because we are running the data through another program. I agree with your rewording, it is definitely more clear :)